packages/components/src/components/form/Range/Range.tsx

```diff
@@ -1,44 +1,35 @@
-import React from 'react';
+import React, { useLayoutEffect, useMemo, useRef, useState } from 'react';
 import styled, { css, CSSObject } from 'styled-components';
+import { variables } from '@trezor/components';
 
 const track = css`
     background: ${({ theme }) => theme.BG_GREEN};
-    cursor: pointer;
     height: 3px;
 `;
 
 const thumb = css`
     appearance: none;
     background: ${({ theme }) => theme.TYPE_WHITE};
     border-radius: 50%;
-    box-shadow: 0 2px 4px 2px rgba(0, 0, 0, 0.2);
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.5);
     cursor: grab;
     height: 26px;
     margin-top: -12px;
     width: 26px;
 
     :active {
-        box-shadow: 0 2px 4px 3px rgba(0, 0, 0, 0.2);
+        box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.5);
         cursor: grabbing;
     }
 `;
 
-type RangeProps = {
-    className?: string;
-    max?: number;
-    min?: number;
-    step?: string;
-    onChange: React.ChangeEventHandler<HTMLInputElement>;
-    thumbStyle?: CSSObject;
-    trackStyle?: CSSObject;
-    value: number;
-};
-
 const Input = styled.input<Pick<RangeProps, 'thumbStyle' | 'trackStyle'>>`
     appearance: none;
-    margin: 30px 0 12px 0;
+    margin-top: 16px;
+    padding: 14px 0;
```

Comment:
`padding` causes  browser-defined `background-color` to be used, see custom coinjoin strategy in dark mode. `background` should therefore be set to `none` here (not in `AnonymityLevelSlider` as it is now).

Reply:
44f52d3a252c5252508bf88c5c8c979357cd09e9

Reply:
44f52d3a252c5252508bf88c5c8c979357cd09e9